deblur/workflow.py

```diff
@@ -206,9 +206,14 @@ def remove_artifacts_seqs(seqs_fp,
                 # if * means no match
                 if line[1] == '*':
                     continue
-                # check if % identity and coverage are large enough
-                if (float(line[2]) >= sim_thresh*100) and (float(line[13]) >= coverage_thresh*100):
-                    aligned_seq_ids.add(line[0])
+                # check if % identity[2] and coverage[13] are large enough
+                # note e-value is [10]
+                if negate:
```

Comment:
Can you extend/modify the tests so we test both branches in this if? (that is, sometimes with negate=True, and others with negate=False)